bigchaindb/web/views/blocks.py

```diff
@@ -0,0 +1,62 @@
+"""This module provides the blueprint for the blocks API endpoints.
+
+For more information please refer to the documentation on ReadTheDocs:
+ - https://docs.bigchaindb.com/projects/server/en/latest/drivers-clients/
+   http-client-server-api.html
+"""
+from flask import current_app
+from flask_restful import Resource, reqparse
+
+from bigchaindb.web.views.base import make_error
+
+
+class BlockApi(Resource):
+    def get(self, block_id):
+        """API endpoint to get details about a block.
+
+        Args:
+            block_id (str): the id of the block.
+
+        Return:
+            A JSON string containing the data about the block.
+        """
+
+        pool = current_app.config['bigchain_pool']
+
+        with pool() as bigchain:
+            block = bigchain.get_block(block_id=block_id)
```

Comment:
According to [the specification](https://github.com/bigchaindb/bigchaindb/pull/830/files#diff-9cd1814988545a4c7fb7457a3f928312R308) this endpoint should take a query parameter `?status`.

For more detail see spec.